<launch>    
	<!-- Arguments -->
	<arg name="filename" default="$(find occupancy_map_visualizer)/samples/gridmap2d.og2" />

    <!-- Parameters -->
    <param name="fixed_frame_id"   		value="map" />
    <param name="gridmap2d_filename" 	value="$(arg filename)" />

    <!-- Run Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find occupancy_map_visualizer)/rviz/gridmap2d.rviz" />

    <!-- GridMap2D Visualization -->
    <node pkg="occupancy_map_visualizer"	type="gridmap2d_visualizer_node"	name="gridmap2d_visualizer_node"	output="screen"/>
</launch>
